src/pages/staticPages/tools/mutationMapper/MutationMapperTool.tsx

```diff
@@ -573,18 +630,35 @@ export default class MutationMapperTool extends React.Component<
     @autobind
     @action
     protected handleLoadExamplePartiallyAnnotated() {
-        this.inputText = require('raw-loader!./resources/standaloneMutationDataExample.txt');
+        if (this.referenceGenomeSelection === referenceGenome.GRCH37) {
+            this.inputText = require('raw-loader!./resources/standaloneMutationDataExample.txt');
```

Comment:
just rename this file standaloneMutationDataExampleGrch37?  then you can compose it